src/index.ts

```diff
@@ -59,8 +59,8 @@ import { parseServerRequest } from './parse-server-request';
 export { parseServerRequest };
 
 // Custom widget with BaseWidget class
-import { BaseWidget } from './base-widget';
-export { BaseWidget };
+import { BaseWidget, connectNoop } from './base-widget';
+export { BaseWidget, connectNoop };
```

Comment:
do you know what's the reason why we aren't using `export { BaseWidget, connectNoop } from './base-widget';` in this file?